src/adapters/poi/poi.js

```diff
@@ -6,17 +6,19 @@ import ExtendedString from "../../libs/string";
 const ZOOM_BY_POI_TYPES = [{type : 'street', zoom : 17}, {type : 'house', zoom : 19}, {type : 'poi', zoom : 18, panel: true}]
 const DEFAULT_ZOOM = 16
 
+export const POI_TYPE = 'poi'
 
 export default class Poi {
-  constructor(id, name, type, latLon, className, subClassName, tags) {
+  constructor(id, name, alternativeName, type, latLon, className, subClassName) {
     this.id = id
     this.name = name
+    this.alternativeName = alternativeName
     this.type = type
     this.latLon = latLon
     this.className = className
     this.subClassName = subClassName
-    this.tags = tags
-    this.computeZoom()
+    this.zoom = this.computeZoom()
+    this.isFavorite = true
```

Comment:
:thinking: what's the use of this boolean ? and is it always `true` ?

Reply:
DK
it's an artefact comming from my investigation .